src/Form.tsx

```diff
@@ -423,8 +423,14 @@ const _Form: Form = React.forwardRef(
       notify(onSubmitFinished, [err]);
     };
 
+    const clearPendingValidations = () => {
+      flushTimeout.clear();
+      queueRef.current.length = 0;
+    };
+
     const handleSubmit = (e?: React.SyntheticEvent) => {
       if (e && e.preventDefault) e.preventDefault();
+      clearPendingValidations();
```

Comment:
@AaronBuxbaum why do we need to clear pending validations here, if we're going to clear them more-or-less immediately afterward in `submit` below?

Reply:
@jquense suggested adding it in both places in case consumers call submit directly. That said it does seem like it gets resolved with that single usage anyway. I want to check with Jason just in case there's some reason I'm unaware of that we need this line, but it does look like these do end up calling line 442 anyway